src/main/java/frc/robot/subsystems/intake/commands/HoldIntakeInPlace.java

```diff
@@ -17,9 +18,9 @@ public HoldIntakeInPlace() {
     @Override
     public void execute() {
         if (intake.switchedToDefaultCommand() || Robot.justEnabled()) {
-            angle = intake.getAngleMotorAngle();
+            angle = intake.getAngleMotorAngle().getRadians();
             angle = MathUtil.clamp(angle, IntakeConstants.ANGLE_DOWN, IntakeConstants.ANGLE_UP);
         }
-        intake.setAngleMotorAngle(IntakeConstants.ANGLE_UP);
+        intake.setAngleMotorAngle(Rotation2d.fromDegrees( IntakeConstants.ANGLE_UP));
```

Comment:
Instead of casting, `IntakeConstants.ANGLE_UP` should be of type `Rotation2d`. Be consistent in your code, try to minimize the types you use.